File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    const answer = [];
    
    for(let i = 0; i < numbers.length-1; i++){
        let fixed = numbers[i];
        
        for(let j = i+1; j < numbers.length; j++){
            if(!answer.includes(fixed + numbers[j])){
                answer.push(fixed + numbers[j]);
            }
        }
    }
    
    return answer.sort((a,b) => a-b);
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    
    for i in range(len(numbers)-1):
        fixed = numbers[i]
        
        for j in range(i+1, len(numbers)):
            sum = fixed + numbers[j]
            if(sum in answer):
                continue
            else:
                answer.append(sum)
    
    answer.sort()
    return answer